test/libsolidity/syntaxTests/operators/custom/operator_with_different_param_return_types.sol

```diff
@@ -0,0 +1,43 @@
+type Int is int128;
```

Comment:
From the name I expected this to be testing functions returning mutiple values of different types.
`operator_with_different_param_return_types.sol` -> `operator_with_different_parameter_types_different_from_return_types.sol`